### ISSUE TITLE
BR1 - Pause button focus needs improving/No focus on hover for Pause or Prev/Next

### ISSUE THREAD

**Issue by jszasz** [brharp/hjckrrh#117]
It would be nice if the focus on the Pause button was a little easier to see.

**Comment by mmafe** [comment]
No indication when hovering over the banner buttons.

**Comment by mmafe** [comment]
Applied as part of hotfix_20150807.3.0

**Comment by jszasz** [comment]
Confirmed.
